connect/transforms/src/main/java/org/apache/kafka/connect/transforms/InsertField.java

```diff
@@ -128,8 +131,10 @@ public void configure(Map<String, ?> props) {
     @Override
     public R apply(R record) {
         if (operatingSchema(record) == null) {
+            log.trace("Applying InsertField SMT with null schema. Record: {}", record);
             return applySchemaless(record);
         } else {
+            log.trace("Applying InsertField SMT with schema. Record: {}", record);
```

Comment:
Is it clear that the record is the _before_ record? Maybe "Applying InsertField SMT with schema to record {}" instead?